src/sentry/api/serializers/models/exporteddata.py

```diff
@@ -29,6 +30,6 @@ def serialize(self, obj, attrs, user, **kwargs):
             "dateCreated": obj.date_added,
             "dateFinished": obj.date_finished,
             "dateExpired": obj.date_expired,
-            "query": {"type": obj.query_type, "info": obj.query_info},
+            "query": {"type": ExportQueryType.as_str(obj.query_type), "info": obj.query_info},
```

Comment:
The serializer will make sure the frontend receives the same data it should be sending (the string version of `query_type`)